packages/ui/react-ui-table/src/model/table-model.ts

```diff
@@ -490,7 +305,12 @@ export class TableModel<T extends BaseTableRow = { id: string }> extends Resourc
     const { props } = this._projection.getFieldProjection(field.id);
     switch (props.format) {
       case FormatEnum.Ref: {
-        setValue(this._rows.value[rowIdx], field.path, value);
+        // TODO(ZaymonFC): This get's called an additional time by the cell editor onBlur, but with the cell editors
+        // plain string value. Maybe onBlur should be called with the actual value?
+        if (!isReactiveObject(value)) {
```

Comment:
nit: to avoid extract break, negate the if and put the setValue inside (slightly clearer)